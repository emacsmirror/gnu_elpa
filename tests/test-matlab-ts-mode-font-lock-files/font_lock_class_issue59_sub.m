% -*- matlab-ts -*-
classdef font_lock_class_issue59_sub < font_lock_class_issue59_super
    methods
        function obj=font_lock_class_issue59_sub
            obj@font_lock_class_issue59_super;
            disp('in SubClass')
        end
    end
end
