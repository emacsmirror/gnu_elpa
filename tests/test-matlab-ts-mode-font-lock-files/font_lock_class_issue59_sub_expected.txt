c ccc ccccccccc ccc
kkkkkkkk fffffffffffffffffffffffffff o fffffffffffffffffffffffffffff
    kkkkkkk
        kkkkkkkk vvvofffffffffffffffffffffffffff
            dddwwwwwwwwwwwwwwwwwwwwwwwwwwwwwwD
            BBBBbSss ssssssssSb
        kkk
    kkk
kkk
